src/common/pre-packaging-helper.sh

```diff
@@ -34,6 +34,27 @@ setup_maven(){
   export PATH=$M2_HOME/bin:$PATH
 }
 
+
```

Comment:
Shouldn't this be a node-helper? pre-packaging is too generic.
Shouldn't the apiserver use this too?